src/test/java/apicurito/tests/configuration/templates/ApicuritoOperatorhub.java

```diff
@@ -0,0 +1,74 @@
+package apicurito.tests.configuration.templates;
+
+import io.syndesis.qe.marketplace.manifests.Bundle;
+
+import org.json.JSONArray;
+import org.json.JSONObject;
+
+import java.util.concurrent.TimeUnit;
+
+import apicurito.tests.configuration.Component;
+import apicurito.tests.configuration.ReleaseSpecificParameters;
+import apicurito.tests.configuration.TestConfiguration;
+import apicurito.tests.steps.ConfigurationOCPSteps;
+import apicurito.tests.utils.openshift.OpenShiftUtils;
+import io.fabric8.kubernetes.client.dsl.base.CustomResourceDefinitionContext;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+
+public class ApicuritoOperatorhub extends ApicuritoInstall {
+    public static final String SUBSCRIPTION_NAME = "fuse-apicurito";
+
+    public static void deploy(String updateChannel, String crName) {
+        log.info("Deploying using Operatorhub");
+
+        Bundle.createSubscription(ConfigurationOCPSteps.getOpenShiftService(), SUBSCRIPTION_NAME,
```

Comment:
So it is not possible or very time demanding to get rid of marketplace-utilities?